app/pages/news/_slug.vue

```diff
@@ -67,6 +67,13 @@ export default {
       }
       const shortenMeta = this.stripHTMLTags(this.selectedNews?.excerpt?.rendered)
       return this.truncate(shortenMeta, 150)
+    },
+    metaOgImage () {
+      if (!this.selectedNews) {
+        return this.metaImage
+      }
+      const metaImageACF = this.selectedNews?.acf?.ogimage
+      return metaImageACF || this.metaImage
```

Comment:
This `||` notation in JS drives me crazy :)
Everytime I see it, I need to remind myself that it returns actual values and not a Boolean one. I guess it's different for Typescript.